lib/patch/file-reader.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var utils = require('../utils');
+
+function apply() {
+  utils.patchClass('FileReader');
```

Comment:
Quick review, not much time this week:
- What if the patch has been applied in `lib/patch/event-target.js` already ?
- `patchClass('FileReader')` will `new FileReader()`, is that allowed ?